mithril-client-cli/src/commands/cardano_db/download.rs

```diff
@@ -361,6 +363,19 @@ impl PreparedCardanoDbDownload {
             );
         }
 
+        if include_ancillary {
+            println!(
+                r###""Upgrade and replace the restored ledger state snapshot to LMDB flavor by running the command:
```

Comment:
There is an extra `"` before the Upgrade:
```suggestion
                r###"Upgrade and replace the restored ledger state snapshot to LMDB flavor by running the command:
```